MaxText/layers/attentions.py

```diff
@@ -1262,8 +1280,8 @@ def __call__(
       query = self.query_projection(inputs_q)
       key = self.kv_projection(inputs_kv, proj_name="key")
       value = self.kv_projection(inputs_kv, proj_name="value")
-
-      if self.use_qk_norm:
+      is_llama4_decoder_block = self.config.decoder_block == "llama4"
+      if self.use_qk_norm and not is_llama4_decoder_block:
```

Comment:
Can we handle both llama4 and Gemma 3 use case here?

e.g. 

if self.use_qk_norm: 
   if is_llama4_decoder_block:
      Do Llama4 norm
    else:
       Do default